native/packages/react-native-bpk-component-button-link/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import BpkButtonLink from './src/BpkButtonLink';
```

Comment:
We should export the flow types too

Reply:
I was just looking at all the other packages and we're not exporting flow types anywhere else, shall I start from this package?

Reply:
Here is an example -> https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-chip/index.js